.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [ 1.17.x ]
+    runs-on: ubuntu-latest
```

Comment:
do we need to specify zendesk runners?

Reply:
since this is a public repo we can use the normal runners

Reply:
But we're using GitHub Actions such as `zendesk/setup-go@v2` already